app/javascript/src/components/Dashboard/Tasks/NewTaskForm.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import * as yup from "yup";
+import { Formik, Form } from "formik";
+import { Input, Textarea, Select } from "neetoui/formik";
+import { Button, Switch } from "neetoui";
+
+const tagValues = [
+  { value: "Internal", label: "Internal" },
+  { value: "Agile Workflow", label: "Agile Workflow" },
+  { value: "Bug", label: "Bug" },
+];
+
+const contacts = [
+  { value: "Karthik Menon", label: "Karthik Menon" },
+  { value: "MS Dhoni", label: "MS Dhoni" },
+];
```

Comment:
I feel that it's always better to move out any constants to a separate independent file like `constants.js`.